test/functional/element_creation.js

```diff
@@ -69,5 +69,67 @@ describe('element creation', () => {
     var el = container.childNodes[0];
     expect(el.id).to.equal('test');
   });
+
+  describe('for HTML elements', () => {
+    it('should use the xhtml namespace', () => {
+      patch(container, () => {
+        elementVoid('div');
+      });
+
+      var el = container.childNodes[0];
+      expect(el.namespaceURI).to.equal('http://www.w3.org/1999/xhtml');
```

Comment:
it scares the crap out of me that this just happens to be the default. Anyway we can check that this element was created using createElement VS. createElementNs? If yes, lets add a check.